### PR TITLE
Feature/async post

### PR DESCRIPTION
**Pull request by DeviantBadge**
Added async methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ Kotlin DSL http client
 
 gradle kotlin DSL:
 ```kotlin
-compile(group = "io.github.rybalkinsd", name = "kohttp", version = "0.8.0")
+implementation(group = "io.github.rybalkinsd", name = "kohttp", version = "0.8.0")
```
Comment (rybalkinsd) [inline README.md]: already exists in master

File: build.gradle.kts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    kotlin("jvm") version "1.3.20"
+    kotlin("jvm") version "1.3.31"
```
Comment (rybalkinsd) [inline build.gradle.kts]: 1.3.40

File: src/main/kotlin/io/github/rybalkinsd/kohttp/ext/StringExt.kt [hunk 3]
```diff
@@ -61,8 +61,19 @@ fun String.httpGet(client: Call.Factory = defaultHttpClient): Response =
  * @since 0.1.0
  * @author sergey
  */
-fun String.asyncHttpGet(client: Call.Factory = defaultHttpClient): Deferred<Response> =
-    GlobalScope.async(context = Unconfined) {
-        client.suspendCall(Request.Builder().url(this@asyncHttpGet).build())
+fun String.httpGetAsync(client: Call.Factory = defaultHttpClient): Deferred<Response> =
+    GlobalScope.async(context = Dispatchers.Unconfined) {
+        client.call(Request.Builder().url(this@httpGetAsync).build())
```
Comment (rybalkinsd) [inline src/main/kotlin/io/github/rybalkinsd/kohttp/ext/StringExt.kt]: why did we change `suspendCall` to `call` invocation`?

Comment (DeviantBadge) [reply src/main/kotlin/io/github/rybalkinsd/kohttp/ext/StringExt.kt]: Changed back

File: build.gradle.kts (hunk 2 above)
Comment (DeviantBadge) [reply build.gradle.kts]: Updated

**Pull request merged**